web/beacon-app/src/features/auth/routes/LoginPage.tsx

```diff
@@ -51,20 +62,53 @@ export function Login() {
   }, [param?.accountVerified]);
 
   useEffect(() => {
-    if (!isAuthenticated(login)) {
+    if (!isAuthenticated(authenticate)) {
       clearSessionStorage();
     }
-  }, [login]);
+  }, [authenticate]);
 
   useEffect(() => {
-    if (login.authenticated) {
-      const token = decodeToken(login?.auth?.access_token) as any;
-      Store.setAuthUser(token, !!login.authenticated);
+    if (authenticated) {
+      setCurrentUserEmail('');
+      const token = decodeToken(auth?.access_token) as any;
+      Store.setAuthUser(token, !!authenticated);
       removeCookie('invitee_token');
       navigate(APP_ROUTE.DASHBOARD);
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [login.authenticated, navigate, login?.auth?.access_token]);
+  }, [authenticated, navigate, auth?.access_token]);
+
+  useEffect(() => {
+    if (error && error.response.status === 403) {
+      // if error is 400 display a toast with a resend email button
+      toast.error(
+        <div>
+          <p>
+            <Trans>Please verify your email address and try again!</Trans>
+          </p>
+          <div className="align-center items-center justify-center">
```

Comment:
Would you mind checking the style for the button? It doesn't look centered in the below screenshot, unless the intention is to have it line up with the beginning of the text above it.

<img width="377" alt="Screenshot 2023-09-22 at 1 15 02 PM" src="https://github.com/rotationalio/ensign/assets/94616884/e3002f45-3eb5-4e17-87bd-6f65b1a44bd8">

Reply:
Ultimately, the goal was to align it with the other content. I removed `text-center` attribute but overlooked removing the remaining elements.